client/components/learn/Resources/resources-model.ts

```diff
@@ -181,6 +196,11 @@ export const RESOURCES: Record<ResourceCategory, ResourceDescription[]> = {
       url: 'https://javascript30.com/',
       cover: `${process.env.CLOUDFRONT_PUBLIC}/public/images/resources/javascript30_500w.png`,
     },
+    {
+      title: 'Frontend Mentor',
```

Comment:
Don't know much about them so I don't feel comfortable adding it to the list yet. I'd have to get familiar with them first.

For the moment let's drop it.